blog/2024/10/julia-1.11-highlights.md

```diff
@@ -0,0 +1,124 @@
++++
+mintoclevel = 2
+maxtoclevel = 3
+title = "Julia 1.11 Highlights"
+authors = "The Julia contributors"
+published = "1 Oct
+rss_pubdate = Date(2024, 10, 01)
+rss = """Highlights of the Julia 1.11 release."""
++++
+
+...
+
+
+The full list of changes can be found in the [NEWS file](https://github.com/JuliaLang/julia/blob/release-1.11/NEWS.md), but here we'll give a more in-depth overview of some of the release highlights.
+
+\toc
+
+# `Array` now implemented in Julia, new `Memory` type
+*Jameson Nash* , *Oscar Smith*
+
+
+# New `public` keyword
+*Lilith Hafner*
+
+
+# `ScopedValue`
+*Valentin Churavy*
+
+
+# Manifest versioning
+*Ian Butterworth*
+
+# Improved tab completion and hinting in the REPL
+*Ian Butterworth*, *Shuhei Kadowaki*
+
+Tab completion has become more powerful in 1.11 and gained hinting when there is a singular completion available that can be completed with tab.
+
+(Embed a video like this:)
+https://github.com/JuliaLang/julia/assets/1694067/6179d799-63f1-47dc-abe2-ac10abb29bc9
+
+
+If you prefer to not have hinting enabled, disable it via your `startup.jl` with
+```
+atreplinit() do repl
+    if VERSION >= v"1.11.0-0"
+        repl.options.hint_tab_completes = false
+    end
+end
+```
+
+
+# Sources section in Project.toml in Pkg.jl
+*Kristoffer Carlsson*
+
+# Precompile file relocatability
+*Florian Atteneder*
+
+Enabling relocation of cache files, together with other improvements coming in v1.11,
+paves the way for Pkg.jl to serve cache files in the future.
```

Comment:
```suggestion
will help Pkg.jl to serve cache files in the future.
```

Reply:
I don't think this document is read for line by line comments. Or just push the fixes in that case, it will be a bit too noisy with comments otherwise I think.